src/release-notes/index.md

```diff
@@ -6,7 +6,80 @@ next: false
 
 # Release Notes
 
-## Version 1.3 (2024-06-05)
+## Version 1.3.1 (YYYY-MM-DD)
+
+### Main Changes
+
+- Added the Two-Factor Authentication (2FA) support.
+
+  > Documented here:\
+  > <!-- TODO: add links to new tutorial and the dedicated 2FA subtopic -->
+
+- Added the ability to select the destination countries when downloading fraud data via FIB Web App.
+
+  > Documented here:\
+  > [Tutorials: Web App > Downloading fraud data](../tutorials-web/downloading-fraud-data.md)
+
+- Added the ability to split the reward for uploaded contributions between the contributor and the original source of the fraud event data.
+
+  > Documented here:\
+  > [Overview > Tokenomics](../overview/tokenomics.md#reward-split) — **Reward split** subtopic added.
+
+- The expiry time of any given contribution now depends on its fraud type.
+
+  > Documented here:\
+  > [Overview > Contributions > Retrieving contributions](../overview/tokenomics.md#reward-split) — `expiryDate` description updated with details.
+
+- Multiple updates to the FIB Web App UI, including:
+  - Added shortcuts for convenient timespans (e.g., one week, one month, etc.) when selecting a date or range of dates.
+  - Made the process of removing records selected for upload straightforward and convenient.
+  - Error messages are expanded with details and are more explicit now.
+  - When downloading fraud data, the result now contains exhaustive filter values.
+  - Various text improvements throughout the UI.  
+
+### API Changes
```

Comment:
I believe date-time format change should be reflected listing all calls affected, please check where we have date-time data among described API calls